src/components/admin.tsx

```diff
@@ -1044,6 +1056,11 @@ function Admin({ user }: adminProps) {
                       key={`navbar-${addressElement.postalcode}`}
                     >
                       <Container fluid className="justify-content-end">
+                        {addressElement.assigneeCount > 0 && (
+                          <Badge bg="danger" pill>
```

Comment:
![Screenshot 2022-12-12 at 2 12 04 PM](https://user-images.githubusercontent.com/40650158/206977465-ffac4633-dfdd-49e1-93cf-6809f27d831f.png)
I think we can improve the UX on this led indicator. On the admin lvl, It seems to indicate the count for the "personal" slip rather than actual HH assignments.
How about putting this badge within its corresponding button ?

Reply:
A second badge is possible. It can be placed besides its button.

The badge appears within a Button. But badges refuse to appear within a DropDownButton.

Reply:
Maybe can google or stackoverflow this.
Doesn't necessarily have to be a "badge". More of UX flow thing. 
Putting it outside the btn can raise the question of whether they are related or not. When its inside, then it's almost self explanatory.

Reply:
![Screenshot 2022-12-13 at 5 24 27 PM](https://user-images.githubusercontent.com/40650158/207279010-5469a833-2dd8-406f-8462-149169b3f08a.png)
This will be the ideal UX